inspectit-oce-bootstrap/build.gradle

```diff
@@ -1,7 +1,5 @@
 dependencies {
-    implementation(
-            'io.opencensus:opencensus-impl:0.18.0'
-    )
+   //No bootstrap dependencies so far...
```

Comment:
should we at least have a `compileOnly` on the opencensus-api? I would prefer that actually? then you can compile against OC..